feat: contribution guidelines (#607)
ory/web: src/markdown/pages/contribute.md

---
published: true
path: '/contribute/'
title: 'Contribution Guidelines'
metaTitle: 'Ory Contribution Guidelines'
metaDescription: 'Read the Ory contribution guidelines.'

> lastUpdatedAt: 'July 25, 2022'

Reviewer: ```suggestion
lastUpdatedAt: 'March 28, 2022'
```

https://github.com/ory/web/blob/master/src/markdown/pages/contribute.md

-->

<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->

- [Introduction](#introduction)
- [Frequently asked questions](#frequently-asked-questions)
- [How can I contribute?](#how-can-i-contribute)
- [Communication](#communication)
- [Contributing Code](#contributing-code)
- [Documentation](#documentation)
- [Disclosing vulnerabilities](#disclosing-vulnerabilities)
- [Code Style](#code-style)
  - [Working with Forks](#working-with-forks)
- [Conduct](#conduct)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

## Introduction

There are many ways in which you can contribute, beyond writing code. The goal
of this document is to provide a high-level overview of how you can get
involved.

_Please note_: We take Ory's security and our users' trust very
seriously. If you believe you have found a security issue in Ory, please
responsibly disclose by contacting us at security@ory.sh.

First: As a potential contributor, your changes and ideas are welcome at any

> hour of the day or night, weekdays, weekends, and holidays. Please do not ever
> hesitate to ask a question or send a pull request.

Reviewer: ```suggestion
hour of the day or night, weekdays, weekends, and holidays. Please do not
hesitate to ask questions or send a pull request.
```

> If you are unsure, just ask or submit the issue or pull request anyways. You

Reviewer: ```suggestion
If you are unsure, just ask or submit the issue or pull request either way. You
```

won't be yelled at for giving it your best effort. The worst that can happen is

> that you'll be politely asked to change something. We appreciate any sort of

Reviewer: ```suggestion
that you'll be asked to change something. We appreciate any sort of
```

contributions, and don't want a wall of rules to get in the way of that.

> That said, if you want to ensure that a pull request is likely to be merged,

Reviewer: ```suggestion
If you want to ensure that a pull request is likely to be merged,
```

> talk to us! You can find out our thoughts and ensure that your contribution

Reviewer: ```suggestion
talk to us! We will ensure that your contribution
```
